activestorage/CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+*   Error entries appear in the log when we request ActiveStorage controllers
+    (`ActiveStorage::Representations::ProxyController#show`,
+    `ActiveStorage::Blobs::ProxyController#show`).
+    
+    These entries look like:
+    ```text
+    Couldn't find template for digesting: active_storage/representations/proxy/show
+    ```
+    
+    There is the issue about it
+    ([#39286](https://github.com/rails/rails/issues/39286)). This change fixes it. 
+    
+    These controllers use the method
+    `ActionController::ConditionalGet#http_cache_forever`,
+    and therefore `ActionController::ConditionalGet#combine_etags`
+    method, and therefore `ActionController::EtagWithTemplateDigest` module via
+    `etaggers` array. `ActionController::EtagWithTemplateDigest` module requires
+    a template (view). We have no views in ActiveStorage, so
+    `EtagWithTemplateDigest` is now turned off in ActiveStorage controllers by
+    `etag_with_template_digest` class attribute.   
+
+    *Evgeniy Rashchepkin*
+
```

Comment:
These controllers are new in master, so no changelog entry is needed.

Reply:
OK, I have removed the changelog entry.